src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,157 @@
+package duke;
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+
+/**
+ * Stores in real-time the edits to the TaskList directly into the destination text file.
+ */
+public class Storage {
+    private String filepath;
+
+    /**
+     * Creates new Storage object.
+     *
+     * @param filepath Filepath of text file to store information in.
+     */
+    public Storage(String filepath) {
+        this.filepath = filepath;
+        makeFile();
+        load();
+    }
+
+    /**
+     * Creates a folder and destination file if either of them do not exist.
+     */
+    public void makeFile() {
+        File textFile = new File("data/duke.txt");
+        Path path = Paths.get("data");
+        if (!Files.isDirectory(path)) {
+            try {
+                Files.createDirectories(path);
+            } catch (IOException e) {
+                System.err.println("Failed to create directory!" + e.getMessage());
+            }
+        }
+        if (!textFile.exists()) {
+            try {
+                textFile.createNewFile();
+            } catch (IOException e) {
+                System.err.println("Failed to create file!" + e.getMessage());
+            }
+        }
+    }
+
+    /**
+     * Reads the text file and stores the String commands line by line into an array.
+     *
+     * @return String array containing commands separated by lines.
+     */
+    public String[] load() {
+        File textFile = new File("data/duke.txt");
+        Integer counter = 0;
```

Comment:
I believe you can use int to access array indexes feel free to correct me if I'm wrong